docs/docs/guides/guidelines.md

```diff
@@ -0,0 +1,229 @@
+
+### Do:
+
+- Make sure to read every available reply.
+- Think about which reply best satisfies the request of the user.
+- Rank replies based on how well they adhere to the guidelines. Factual accuracy
+  and helpfulness are first and foremost.
+- Penalize replies that fail to provide adequate warnings or caveats.
+- Penalize replies that are difficult to read due to a lack of formatting,
+  capitalization or other errors.
+- Penalize replies if the requested information is obfuscated by superfluous
+  details that make up a large part of the message.
+
+### Don't:
+
+- Rank replies based on personal beliefs. Assuming an opinion was warranted,
+  fulfills the users request and doesn't violate any guidelines, it should not
+  impact the rating of the reply.
+- Rank replies based on how long and short they are - instead, find out which
+  reply best answers the query of the user.
+
+## 7. Label Explanation {#label-explanation}
+
+**Is the message spam?**
+
+Is the message an unrelated piece of text that doesn't contribute to the
+previous prompt in any way?
+
+**Is it a bad reply, as an answer to the prompt task?**
+
+Does the assistant reply fail to engage the request of the user or does it
+contain incorrect information?
+
+### Tags
+
+**Not Target Language**: Unless specifically requested, is the language used
+different from the target language of the data set?
+
+**Not Appropriate**: In case of a user prompt, is it missing a clear request?
+
+**Contains PII**: Does the message contain personally identifying information
+(PII) that isn't publicly available and can be used to determine the identity of
+the user or a private third party?
+
+**Hate Speech**: Does the message contain prejudice towards a group classified
+as protected according to German, UK or US law?
+
+**Sexual Content**: Does the message contain sexually explicit content, such as
+descriptions of sexual acts?
+
+### Scales
+
+**Low Quality/High Quality**: How well does the message adhere to the
+guidelines? For assistant replies, factual accuracy and helpfulness are first
+and foremost.
+
+**Serious/Humorous**: Does the message contain sarcasm, a play on words or other
+humorous embellishments?
+
+**Ordinary/Creative**: How unique is the message? Is the user asking a simple
+question, or is it challenging the assistant with a problem that requires
+creative thinking? Is the assistant reply straightforward, or does it contain a
+unique approach to a problem?
+
+**Rude/Polite**: How polite is the message towards the recipient? Does it use
+words like "please" or is it dismissive, rude or insulting towards the other
+side of the conversation?
+
+**Violent/Harmless**: Does the message contain descriptions of explicit violent
+acts? Is it glorifying, encouraging or downplaying violent behavior?
+
+---
+
+We drafted this guide using ChatGPT, OpenAI’s large-scale language-generation
+model. Upon generating draft language, the authors reviewed, edited, and revised
```

Comment:
What specifically is meant by a use of ChatGPT? Maybe rephrase it to "We used ChatGPT as one of the tools for drafting this guide"?